guides/common/modules/con_hammer-cli-overview.adoc

```diff
@@ -0,0 +1,25 @@
+[id="Hammer-CLI-Overview_{context}"]
+= Hammer CLI overview
+
+You can configure and manage your {ProjectServer} with CLI commands by using Hammer.
+
```

Comment:
When thinking of paragraphs, I don't think of how many sentences that para has, but how many ideas it conveys. So roughly 1 idea = 1 para.

* In this case, the first paragraph is an introduction to the module (because every [module needs an intro](https://redhat-documentation.github.io/modular-docs/#creating-concept-modules)).
* The second paragraph (line 6) marks the beginning of the actual contents of the module by introducing a bulleted list.

So while seeing a single-sentence paragraph makes me somewhat uncomfortable too, that "1 idea = 1 paragraph" guideline takes precedence. Especially here, if one tries to adhere to the mod docs templates (which I do because they are my babies).

Reply:
works for me!